CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - Ansible based operators now gather and serve metrics about each custom resource on port 8686 of the metrics service. ([#1723](https://github.com/operator-framework/operator-sdk/pull/1723))
 - Added the Go version, OS, and architecture to the output of `operator-sdk version` ([#1863](https://github.com/operator-framework/operator-sdk/pull/1863))
 - Added support for `ppc64le-linux` for the `operator-sdk` binary and the Helm operator base image. ([#1533](https://github.com/operator-framework/operator-sdk/pull/1533))
+-  Added `Operator Version: X.Y.Z` information in the operator logs.([#1953](https://github.com/operator-framework/operator-sdk/pull/1953))
```

Comment:
since we have the release of 0.11.
In order, we are able to merge its required: rebase the PR with the master branch and move this entry for the UNRELEASED section. Could you help us with @drnic?

Reply:
@camilamacedo86 ok, done

Reply:
HI @drnic, 

I think maybe you forget to push the change.  This entry needs to be moved for [here](https://github.com/operator-framework/operator-sdk/blob/1ec794c70df839117204c139a1aa9ed98c465d6a/CHANGELOG.md#added).

And really tks for your contribution 🥇 
iIt is very important to us.

Reply:
Oh you released 0.11 without this PR. Sorry.

Reply:
I kept the PR as two commits and fixed/updated your CHANGELOG update commit.